assets/app/view/game/corporation.rb

```diff
@@ -81,7 +81,10 @@ def render
 
         extras = []
         if @game.corporation_show_loans?(@corporation)
-          extras.concat(render_loans) if @game.total_loans&.nonzero?
+          if @game.total_loans&.nonzero?
```

Comment:
In general positive? is better to use because non-zero may be -1. Obviously in this case, -1 would be  bug in itself.